content/community/roadmap/_index.en.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Roadmap for Altinn
+linktitle: Roadmap
+description: High level roadmap and functional description for Altinn.
+weight: 900
+aliases:
+- /roadmap/
```

Comment:
We should use aliases only when moving pages inside docs.altinn.studio, never for new pages.